src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts

```diff
@@ -146,9 +154,19 @@ export const thorchainApi: SwapperApi = {
         const publicClient = viemClientByChainId[chainId as EvmChainId]
         assert(publicClient !== undefined, `no public client found for chainId '${chainId}'`)
 
+        const expectedAmountOut =
+          BigInt(longtailData?.longtailToLExpectedAmountOut ?? 0) || BigInt(0)
```

Comment:
Right side is useless here but I get the paranoia

Reply:
You're right, not really needed - removed as a part of https://github.com/shapeshift/web/pull/6052/commits/436e8b7537e640938e5900bcec78221090702f70.